components/fxa-client/src/fxa_msg_types.proto

```diff
@@ -22,6 +22,11 @@ message AccessTokenInfo {
     required uint64 expires_at = 4;
 }
 
+message IntrospectInfo {
```

Comment:
@eoger please remind me, does the name IntrospectInfo form part of the public API used by consumers? If so, I wonder if we should come up with a more descriptive name, because "introspect" is really an implementation detail that callers shouldn't need to know about. Maybe AuthorizationStatusInfo or similar?

Reply:
Does the consumer actually needs the `token_type` since we know we will always introspect the refresh token? This would simplify the API by simply returning a boolean? What do you think @rfk ?

Reply:
I'm [not excited about a boolean](https://github.com/mozilla/application-services/issues/1263#issuecomment-500577431) for this, but as always I could be convinced.